recvalsite/settings.py

```diff
@@ -33,6 +33,7 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = config("DEBUG", default=False, cast=bool)
+DEBUG_DJANGO = ""
```

Comment:
```suggestion
DEBUG_DJANGO = config("DEBUG_DJANGO", default=DEBUG, cast=bool)
```

?

You might also want to rename this to `USE_DJANGO_DEBUG_TOOLBAR`